Sources/app.js

```diff
@@ -102,16 +146,19 @@ function APIRequest(jsonObj) {
             if (!settings.poll_status_url) return;
         }
 
-        let url    = do_status_poll ? settings.poll_status_url : settings.request_url;
-        let method = (do_status_poll ? settings.poll_status_method : settings.request_method) || 'GET';
+        let url = do_status_poll ? settings.poll_status_url : request_url;
+        let method =
+            (do_status_poll ? settings.poll_status_method : request_method) ||
+            'GET';
```

Comment:
style: please don't remove whitespace, or add wrapping like this - I personally prefer the visual aesthetic.